### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Rusty89**
Merging testing back to develop to update and fix and couple bugs (typos causing build to fail).

### CODE REVIEW

File: client-app/src/app/activity-form/Activity-form.component.ts [hunk 1]
```diff
@@ -4,8 +4,8 @@ import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-activity-form',
-  templateUrl: './activity-form.component.html',
-  styleUrls: ['./activity-form.component.css']
+  templateUrl: './Activity-form.component.html',
+  styleUrls: ['./Activity-form.component.css']
```
Comment (RialJohnson) [inline client-app/src/app/activity-form/Activity-form.component.ts]: Why did you change these back to capital?

File: client-app/src/app/info-form/Info-form.component.spec.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
-import { InfoFormComponent } from './info-form.component';
+import { InfoFormComponent } from './Info-form.component';
```
Comment (RialJohnson) [inline client-app/src/app/info-form/Info-form.component.spec.ts]: Why did you change these back to capital?

File: client-app/src/app/activity-form/Activity-form.component.ts (hunk 1 above)
Comment (Rusty89) [reply client-app/src/app/activity-form/Activity-form.component.ts]: The build on travis gave errors when it was lower case,  All four files within activity form were uppercase.  Without this the build failed.  We could go the other way and make every instance where those files are called lower case, this was just the quick fix.

Comment (Rusty89) [reply client-app/src/app/activity-form/Activity-form.component.ts]: Info-form.components also had similar inconsitencies.
So I just made them consistent.

Comment (Rusty89) [reply client-app/src/app/activity-form/Activity-form.component.ts]: I know the build works locally, but travis was spitting this error.  
ERROR in : Couldn't resolve resource ./activity-form.component.css relative to /home/travis/build/Rusty89/SeniorCenterManagmentStudio/client-app/src/app/activity-form/Activity-form.component.ts

Comment (Rusty89) [reply client-app/src/app/activity-form/Activity-form.component.ts]: @RialJohnson

Comment (RialJohnson) [reply client-app/src/app/activity-form/Activity-form.component.ts]: I'd rather fix them to be all lowercase and maintain this consistency in the future.

**Review by RialJohnson — COMMENTED** [review]
Appreciate you fixing this!
